gusb/gusb-context.c

```diff
@@ -528,12 +530,14 @@ g_usb_context_enumerate (GUsbContext *context)
 		return;
 
 	g_usb_context_rescan (context);
+#ifndef __FreeBSD__
 	if (!libusb_has_capability (LIBUSB_CAP_HAS_HOTPLUG)) {
```

Comment:
I think FreeBSD should implement `libusb_has_capability()` -- it's either API compatible with libusb-1.0 or it's not....

Reply:
Reported on FreeBSD bugzilla: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=224454

Reply:
The FreeBSD developers try to keep up with what "official" libusb is implementing. Instead of checking ifdef __FreeBSD__ it might be worth tracking down by using: 

#define LIBUSB_API_VERSION 0x01000102

Instead.

Reply:
I switched to `has_header_symbol`